assets/sass/components/global/_googlesitekit-table.scss

```diff
@@ -180,6 +183,13 @@
 			&:hover {
 				background-color: rgba($c-secondary, 0.05);
 			}
+
+			&--no-data {
+
+				&:hover {
+					background-color: transparent;
```

Comment:
Moved it down here alone `--with-list` because this is where the original background color is being added.